src/Projects/Modules/Crunchyroll/Clients/Module.Crunchyroll.Avalonia/ViewModels/SeriesDetailViewModel.cs

```diff
@@ -43,19 +70,35 @@ private void OnSelectedItemChanged()
         }
 
 
-        public SeriesDetailViewModel(CrunchyrollService.CrunchyrollServiceClient client)
+        public SeriesDetailViewModel(CrunchyrollService.CrunchyrollServiceClient client,
+            IImageDownloadService imageDownloadService, SearchResponse.Types.SearchItem model)
         {
             this.client = client;
+            this.imageDownloadService = imageDownloadService;
+            this.model = model;
+
+            this.imageDownloadService
+                .DownloadImageAsync(this.model.PortraitImage?.Full ??
+                                    this.model.LandscapeImage?.Large ??
+                                    "https://media.wired.com/photos/5a0201b14834c514857a7ed7/master/pass/1217-WI-APHIST-01.jpg",
```

Comment:
more usage of that magic "missing image" thumbnail, maybe a global constant would be the right choice here